cmake/P4CUtils.cmake

```diff
@@ -141,13 +141,28 @@ endmacro(p4c_add_test_list)
 
 # generate a list of test name suffixes based on specified testsuites
 function(p4c_find_test_names testsuites tests)
-  foreach(ts "${testsuites}")
+  set(__tests "")
+  p4c_sanitize_path("${testsuites}" abs_paths)
+  foreach(ts ${abs_paths})
     file (GLOB __testfiles RELATIVE ${P4C_SOURCE_DIR} ${ts})
     list (APPEND __tests ${__testfiles})
   endforeach()
   set(${tests} "${__tests}" PARENT_SCOPE)
 endfunction()
 
+# convert the paths from a list of input files to their absolute value,
+# does not follow symlinks.
```

Comment:
does this work when someone uses a symlink in the extensions folder?

Reply:
yes.